src/internal/cluster/zarf.go

```diff
@@ -98,3 +99,41 @@ func (c *Cluster) RecordPackageDeployment(pkg types.ZarfPackage, components []ty
 
 	c.Kube.CreateOrUpdateSecret(deployedPackageSecret)
 }
+
+// SetRegHPAScaleDown enables or disables the HPA scale down for the Zarf Registry.
+func (c *Cluster) EnableRegHPAScaleDown() error {
```

Comment:
The docstring comment title got lost as we were experimenting with function names